synapse/rest/media/v1/_base.py

```diff
@@ -30,6 +30,20 @@
 
 logger = logging.getLogger(__name__)
 
+TEXT_CONTENT_TYPES = [
```

Comment:
can you add a comment saying what this list is used for?

Reply:
(where did this list come from, btw?)

Reply:
> can you add a comment saying what this list is used for?

Absolutely. It's been done and pushed.



> (where did this list come from, btw?)

I got the list from [here](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types), and I quickly plucked the ones out that are _obviously_ text file formats. Fortunately, since it's a hardcoded list, adding ones I forgot/didn't notice and removing ones that I put in there by mistake should be trivial.